app/src/NativeBase/Containers/MyExperienceContainer.tsx

```diff
@@ -0,0 +1,108 @@
+/**
+ * @file the MyExperience screen for the STA Volunteer App
+ */
+import {
+  Heading,
+  VStack,
+  HStack,
+  Checkbox,
+  ScrollView,
+  Flex,
+  Button,
+  Box,
+} from 'native-base'
+import StaRibbon from '../Components/StaRibbon'
+import StaThemeLogo from '@/NativeBase/Assets/Images/Logos/sta-ribbon-logo.svg'
+import FreeSearchBar from '../Components/FreeSearchBar'
+import React from 'react'
+import { RoleGroup, roleGroups } from '@/Services/modules/projects/roleGroups'
+
+const MyExperienceContainer = () => {
+  const [skillsValue, setSkillsValue] = React.useState<string[]>([])
+  const [searchTxt, setSearchTxt] = React.useState<string>('') // search text
+
+  /**
+   * Filter the role groups based on the search text entered by the user
+   */
+  const filteredSkills = roleGroups.filter((roleGroup: RoleGroup) => {
+    return roleGroup.groupName.toLowerCase().includes(searchTxt.toLowerCase())
+  })
+
+  return (
+    <>
+      <HStack marginTop={-10} marginBottom={12} alignItems={'flex-end'}>
+        <Heading size="md" fontWeight="bold" marginRight={-5}>
+          My Experience
+        </Heading>
+        <Flex flex={1} alignItems={'flex-end'}>
+          <StaThemeLogo />
+        </Flex>
+      </HStack>
+      <StaRibbon />
```

Comment:
Great component David. It looks like this may not be needed here for this screen though, as it seems to be replaced with a progress bar in the design.

Reply:
According to Joanna this was to he the STA Ribbon instead which is why I have it included in place of the other object.

Reply:
Just to add to the comment above, Joanna was clear she wanted the STA ribbons included but leaves less room for the progress bar. So need to clarify this point. If using the progress bar, Nativebase has one but looks considerably different to Mia's designs 'out-of-the-box'

Reply:
Ok, cool, thanks David, we'll see where the design goes, in any case great work :)